packages/wordpress-kit/src/lib/tailwindcssPlugin/components/Buttons.ts

```diff
@@ -0,0 +1,96 @@
+const justifications = {
+	'&.is-content-justification-left': {
+		justifyContent: 'flex-start',
+	},
+	'&.is-content-justification-center': {
+		justifyContent: 'center',
+	},
+	'&.is-content-justification-right': {
+		justifyContent: 'flex-end',
+	},
+	'&.is-content-justification-space-between': {
+		justifyContent: 'space-between',
+	},
+};
+
+const buttonWidths = {
+	'&.wp-block-button__width-25': {
+		width: 'calc(25% - 1.5rem * .75)',
+	},
+	'&.wp-block-button__width-50': {
+		width: 'calc(50% - 1.5rem * 0.5)',
+	},
+	'&.wp-block-button__width-75': {
+		width: 'calc(75% - 1.5rem * 0.25)',
+	},
+	'&.wp-block-button__width-100': {
+		width: '100%',
+		'flex-basis': '100%',
+	},
+};
+
+export const ButtonsComponent = ({
+	defaultColor,
+}: {
+	defaultColor: string;
+}) => ({
+	'.wp-block-buttons': {
+		display: 'flex',
+		gap: '1.5rem',
+		flexWrap: 'wrap',
+		alignItems: 'center',
+		'justify-content': 'flex-start',
+		...justifications,
+		'max-width': '650px',
+		'&.alignwide': {
+			maxWidth: '850px',
+		},
+		'&.alignfull': {
+			'@media (min-width:720px)': {
+				// sets a negative margin to allow full width tables to span past the
```

Comment:
```suggestion
				// sets a negative margin to allow full width element to span past the
```

Reply:
Oh, I forgot to change it, nice catch! Done :p